$OpenBSD: patch-src_FbAtoms_hh,v 1.1 2012/01/20 11:53:30 dcoppa Exp $

Add mwm _MOTIF_WM_INFO atom to advertise mwm hints capability: fix
a problem with rxvt-unicode falling out of Fluxbox' control when
started with the borderless option.
(upstream git commit db08329d8bc689ff14cbb9084814d652ae349621)

--- src/FbAtoms.hh.orig	Fri Jan 20 12:29:11 2012
+++ src/FbAtoms.hh	Fri Jan 20 12:30:20 2012
@@ -47,6 +47,7 @@ class FbAtoms { (private)
     FbAtoms();
 
     Atom blackbox_attributes;
+    Atom motif_wm_info;
     Atom motif_wm_hints;
     Atom xa_wm_protocols;
     Atom xa_wm_state;
